package.json

```diff
@@ -4,9 +4,9 @@
   "version": "1.0.0",
   "scripts": {
     "check": "mongodb-js-precommit",
-    "pretest": "mongodb-runner start",
+    "pretest": "mongodb-runner start --topology=replicaset",
     "test": "mocha",
-    "posttest": "mongodb-runner stop",
+    "posttest": "mongodb-runner stop --topology=replicaset",
```

Comment:
Note, setting a global hook for mocha might be the cleanest pattern to follow. Lucas describes how here:
https://www.npmjs.com/package/mongodb-runner